functions/src/mappers/github/pullRequest.mapper.ts

```diff
@@ -23,6 +24,7 @@ export interface GitHubPullRequestModel {
   state: string;
   title: string;
   description: string;
+  number: number;
```

Comment:
I think we should rename this field as it is not clear what it is. I think may `id`?